specs/core/1_custody-game.md

```diff
@@ -479,16 +512,21 @@ def process_challenge_deadlines(state: BeaconState) -> None:
             state.custody_bit_challenge_records.remove(challenge)
 ```
 
+### Miscellaneous processing
+
 In `process_penalties_and_exits`, change the definition of `eligible` to the following (note that it is not a pure function because `state` is declared in the surrounding scope):
 
 ```python
-def eligible(index):
+def eligible(index) -> bool:
     validator = state.validator_registry[index]
     # Cannot exit if there are still open chunk challenges
     if len([record for record in state.custody_chunk_challenge_records if record.responder_index == index]) > 0:
         return False
+    # Cannot exit if there are still open bit challenges
+    if len([record for record in state.custody_bit_challenge_records if record.responder_index == index]) > 0:
+        return False
     # Cannot exit if you have not revealed all of your custody keys
-    elif epoch_to_custody_period(revealer.activation_epoch) + validator.custody_reveal_index <= epoch_to_custody_period(validator.exit_epoch):
+    elif validator.latest_custody_reveal_period <= epoch_to_custody_period(validator.exit_epoch):
```

Comment:
I think if it's _latest reveal_ rather than _next reveal_, the inequality should be <, not  <=. The rationale is that if you exit during epoch N, that means that you used the subkey for epoch N and previous epochs, so you need to reveal all the subkeys up to and including N, so the "latest reveal" should be N for an exit to be valid.